libs/core/src/lib/feed-list-item/components/item/feed-list-item.component.html

```diff
@@ -0,0 +1,91 @@
+<!-- Non mobile view -->
+<div class="fd-feed-list__body" *ngIf="!isMobile">
```

Comment:
Could you try to move some common blocks to separate `ng-template` and use it to build desktop and mobile views? I see we can do it for example for `<p class="fd-feed-list__text">` block. This will make the template more clear and help to maintain the code in the future.

Reply:
Ok, updated